### DISCUSSION TITLE
Automask

### DISCUSSION THREAD

**Discussion by granalberto** [ImageMagick/ImageMagick#3263] [Help]
I'd like some guidelines on what to read for using ImageMagick for getting PNG files with transparent background from a JPG picture of an object. Let's say I take the picture of a watch on a green paper, I want to remove all the green color and make it transparent so I can compose another image by using some form of template.

An example would be great too if that's possible.

**Comment by fmw42** [comment]
If on a Unix-like system, you might try my ImageMagick script, greenscreen, at my web site http://www.fmwconcepts.com/imagemagick

**Reply by granalberto** [reply]
Thank you so much.
